src/site/includes/veteran-navigation.html

```diff
@@ -1,165 +0,0 @@
-<div class="login-container">
```

Comment:
Just pointing out that a file was deleted but this PR's description mentions only updating links.

Reply:
Yeah, maybe it would be better to wait for a site-wide cleanup of Vets.gov assets no longer needed.